foreign_cc/built_tools/cmake_build.bzl

```diff
@@ -6,6 +6,9 @@ def cmake_tool(name, srcs, **kwargs):
     configure_make(
         name="_build_{}".format(name),
         lib_source=srcs,
+        out_binaries = ["cmake"],
```

Comment:
also `cmake.exe` on windows 😅

Reply:
Done - however CMake really needs openssl as a dependency. There are issues with building openssl using these rules at the moment and so supporting a built from source cmake on windows is not something that will currently work.